### PR TITLE
404  and 500 page added

### PR DESCRIPTION
**Pull request by UmbrellaCrow612**
Hi your website doesn't have a base 404 or 500 page so i added them

### CODE REVIEW

File: www/README.md [hunk 1]
```diff
@@ -9,8 +9,7 @@ This is the fresh website source. The fresh website contains:
 
 Start the project:
 
-```
-deno run -A --watch main.ts
-```
+
+deno task start
```
Comment (nikableh) [inline www/README.md]: You accidentally removed \``` ``` around the code block.

Comment (UmbrellaCrow612) [reply www/README.md]: Thanks

**Comment by un-versed** [comment]
Related to https://github.com/denoland/fresh/issues/446

**Comment by UmbrellaCrow612** [comment]
Now the 404 error and 500 error pages work in deployment

**Comment by UmbrellaCrow612** [comment]
i fixed the toc.json spelling mistake, for the word and file, now you won't get a 500 error

**Comment by fdemir** [comment]
adding some styles to 400 and 500 would be cool tho

**Comment by UmbrellaCrow612** [comment]
> adding some styles to 400 and 500 would be cool tho

I would but I just addressed the issue, if you want design something  and I’ll check it out and merge it with mine if it’s suitable.

**Comment by fdemir** [comment]
> I would but I just addressed the issue, if you want design something and I’ll check it out and merge it with mine if it’s suitable.


i think we can move on from another issue after close this pr.

File: docs/toc.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   "concepts": {
     "title": "Concepts",
     "pages": [
-      ["architechture", "Architecture"],
```
Comment (lucacasonato) [inline docs/toc.json]: Please revert this. I'll tackle this later.

**Comment by nikableh** [comment]
@UmbrellaCrow612, please don't make all your contributions in one single PR. That makes it much more difficult to later merge it.

**Comment by UmbrellaCrow612** [comment]
> @UmbrellaCrow612, please don't make all your contributions in one single PR. That makes it much more difficult to later merge it.

What should I do now ?

**Review by nikableh — COMMENTED** [review]
Please don't change already existing files like that. They were fine as is.

**Comment by nikableh** [comment]
> Please don't change already existing files like that. They were fine as is.

For some reason, it doesn't show what I was talking about. I was talking about optimizing the size of the images.

**Comment by nikableh** [comment]
> > @UmbrellaCrow612, please don't make all your contributions in one single PR. That makes it much more difficult to later merge it.
> 
> What should I do now ?

Just wait until your PR is merged, or you are asked to change something. In the meantime, you can work on something else :+1:

**Comment by UmbrellaCrow612** [comment]
> > > @UmbrellaCrow612, please don't make all your contributions in one single PR. That makes it much more difficult to later merge it.
> > 
> > 
> > What should I do now ?
> 
> Just wait until your PR is merged, or you are asked to change something. In the meantime, you can work on something else 👍

Thanks for the advice, it's my first time making a PR 👍

**Review by lucacasonato — COMMENTED** [review]
Please revert all of the changes that are not related to the 404 and 500 pages.

**Comment by UmbrellaCrow612** [comment]
> Please revert all of the changes that are not related to the 404 and 500 pages.

Hi i reverted all the changes not related to the 404 and 500 pages, the deployed version now shows them, do i need to do anything else ?

**Comment by UmbrellaCrow612** [comment]
I reverted all the changes not related to 404 or 500

**Comment by michael-pfister** [comment]
this was left behind a month ago -> new PR #624

**Comment by UmbrellaCrow612** [comment]
Is there an update ?

**Comment by michael-pfister** [comment]
@UmbrellaCrow612 I reworked your PR https://fresh-nmyvm7b07gsg.deno.dev/nonexistentpage

**Comment by UmbrellaCrow612** [comment]
Should I close this one ?

**Comment by michael-pfister** [comment]
@UmbrellaCrow612 yup, checkout #624

**Comment by michael-pfister** [comment]
@UmbrellaCrow612 thanks for collaborating, you did an amazing job 🤝

**Comment by UmbrellaCrow612** [comment]
Not mergers and left behind

**Comment by michael-pfister** [comment]
@UmbrellaCrow612 they might merge #624, I didn't have access to your repo so I had to make a new PR. This can still be considered your work.

**Comment by UmbrellaCrow612** [comment]
It’s fine with me 👍🏻

**Pull request closed without merging**